tests/test_docs.py

```diff
@@ -0,0 +1,57 @@
+import nbformat
+import os
+import pytest
+import re
+import glob
+from nbconvert.preprocessors import ExecutePreprocessor
+
+
+def run_notebook(notebook_path):
```

Comment:
This is a great addition to the tests! Thanks for putting this module together.

Reply:
My only comment is that running it can take a long time. It would be nice to enable developers to skip this test by passing a command line argument.

Reply:
I understand, but we should do testing at the end, and the notebooks are fundamentally the stuff that needs to work. One problem is that *some* notebooks run for a long time (e.g. multi-res), so we should maybe reduce the runtime for testing (but not for generating the docs).

Reply:
Right, my point is that we want everything tested before we merge, so we don't want a time limit for testing. That way when we make a PR Travis will test everything and make sure that all of the tests (including the notebooks) pass. But locally we may prefer to run all of the tests other than the notebooks, so it would be nice to be able to skip this one.

Reply:
Here's where I disagree. The notebooks are the one thing that needs to get tested for API consistency and regression. Travis is OK runtime-wise, at least for now. If you have a single other test that keeps failing, execute it in isolation.

Reply:
I disagree but this discussion can continue on slack outside of this PR.